src/24-uncensorable-wikipedia/index.md

```diff
@@ -0,0 +1,188 @@
+
+We've almost finished creating a web browser extension that will allow you to access IPFS directly from your web browser without installing any additional external tools. Watch https://github.com/ipfs/in-web-browsers for the announcement when we release that browser extension.
+
+## How to Mirror Wikipedia on an IPFS Node
+
+If you want to help make this information available, you can install IPFS on your computer and _"pin"_ the Wikipedia snapshot(s) on that IPFS node. Here are instructions for doing that.
+
+**Prerequisites**:
+* Requires [using the command line](http://lifehacker.com/5633909/who-needs-a-mouse-learn-to-use-the-command-line-for-almost-anything)
+* You need to have enough storage space to hold the snapshot(s) you want to serve.
+    * Turkish Wikipedia is about 15 GB
+* _If possible, use a machine with a public IP address._ If you want to run an ipfs-to-http gateway that lets people access the IPFS content using their web browswer, you need to ensure that youre machine can be reached with a public IP address.
+
+If you don't have enough storage space to hold full copies of the snapshot(s), you can still run an IPFS gateway so that people can rely on you to retrieve the content from the IPFS network on the fly.
+
+**Steps**:
+   1. Install IPFS [following these instructions]()
+   2. Start your IPFS node by running `ipfs daemon` so it can connect to the network.
+   3. Pin the snapshot(s) onto your machine
+      - Pin Turkish Wikipedia: `ipfs pin add tr-wikipedia-ipfs`
+   4. If your machine has a public IP addresses, tell people the address of your gateway. They can use that address to request IPFS content from their web browsers.
+
+**Alternative: Using a Sneakernet to Mirror data**
+
+I'm not kidding. If you'd like to access this content via [sneakernet](https://en.wikipedia.org/wiki/Sneakernet), IPFS is just fine with that. To find out how, follow the instructions in [this tutorial](https://dweb-primer.ipfs.io/avenues-for-access/sneakernets.html)
+
+## How we made this Wikipedia snapshot
+
+The steps we followed to create these first snapshots were:
+
+1. Download the latest snapshot of Wikipedia (in ZIM format) from http://wiki.kiwix.org/wiki/Content_in_all_languages
+2. Unpack the ZIM snapshot using https://github.com/dignifiedquire/zim/commit/a283151105ab4c1905d7f5cb56fb8eb2a854ad67
+3. Configure your IPFS node to enable directory sharding - run `ipfs config --json 'Experimental.ShardingEnabled' true`
+3. Add all the data the node using `ipfs add -w -r --raw-leaves $upacked_wiki`.
+4. Save the last hash of that output. It is the hash of your new Wikipedia snapshot
+
+This work was primarily done by [@kubuxu](https://github.com/kubuxu) [@dignifiedquire](https://github.com/dignifiedquire) and [@lgierth](https://github.com/lgierth) with help from [@whyrusleeping](https://github.com/whyrusleeping). They used code originally written by [@eminence](https://github.com/achin)
```

Comment:
> They used code originally written by [@eminence](https://github.com/achin)

Not sure why the username of the link and the github profile we link to are different, should be the same no?